src/components/BackButton.js

```diff
@@ -12,7 +14,9 @@ const BackButton = ({ url, ...props }) => {
   const history = useHistory();
   return (
     <BackButtonContainer>
-      <img
+      <FaArrowCircleLeft
+        color={props.theme.colors.text}
+        size={50}
         src={backButton}
         alt="navigate to last page"
         onClick={() => history.push(url)}
```

Comment:
can't toggle the colour of an image due to theme changes, so i changed it to an icon.